packages/tallyho/package.json

```
@@ -0,0 +1,71 @@
+{
+  "name": "@web3-onboard/tallyho",
+  "version": "2.0.14",
```

This package can start at 2.0.0 since it is brand new
```suggestion
  "version": "2.0.0",
```